packages/mobile/components/ArticleContent.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from "react";
+import { useState, useEffect } from "react";
+
+import styled from "styled-components";
+import { View, Text, SafeAreaView, Button, TextInput } from "react-native";
+import { ActivityIndicator } from "react-native-paper";
+import moment from "moment";
+import Breadcrumbs from "./Breadcrumbs";
```

Comment:
You removed this file but left the import, please check the app is working when doing a PR